python/tidy/servo_tidy_tests/test_tidy.py

```diff
@@ -266,6 +266,11 @@ def test_file_list(self):
         lst = list(file_list)
         self.assertEqual([os.path.join(base_path, 'whee', 'test.rs')], lst)
 
+    def test_multiline_string(self):
+        errors = tidy.collect_errors_for_files(iterFile('multiline_string.rs'), [], [tidy.check_rust], print_text=True)
+        self.assertEqual(len(list(errors)), 0)
```

Comment:
Shouldn't we be doing `self.assertNoMoreErrors(errors)` here?

Reply:
Good point.  Updated.

Reply:
Thanks. Please squash your commits, and let's land this :)